libraries/Microsoft.Bot.Builder.LanguageGeneration/LGFile.cs

```diff
@@ -132,6 +132,37 @@ public object EvaluateTemplate(string templateName, object scope = null)
             return evaluator.EvaluateTemplate(templateName, scope);
         }
 
+        /// <summary>
+        /// Use to evaluate an inline template str.
+        /// </summary>
+        /// <param name="inlineStr">inline string which will be evaluated.</param>
+        /// <param name="scope">scope object or JToken.</param>
+        /// <returns>Evaluate result.</returns>
+        public object Evaluate(string inlineStr, object scope = null)
```

Comment:
>Evaluate [](start = 22, length = 8)

Why is this in this change?  How does it relate to the lg changes for string interpolation?

Reply:
This was code which was in properties, but should be in LG